appveyor.yml

```diff
@@ -0,0 +1,23 @@
+build: false
+shallow_clone: false
+platform: 'x86'
+clone_folder: c:\projects\symfony
+
+environment:
+  global:
+    PHP: c:/PHP
+
+init:
+  - SET PATH=c:\php\;%PATH%
+
+install:
+  - cinst gow
+  - cinst php
+  - cinst composer
```

Comment:
this does not work, and I think I know why: it uses the windows installer for that, and the windows installer is interactive AFAIK